src/xdocs/config_coding.xml

```diff
@@ -3614,6 +3614,99 @@ class TestHashCode {
       </subsection>
     </section>
 
+    <section name="MatchXpath">
+      <p>Since Checkstyle 8.37</p>
+      <subsection name="Description" id="MatchXpath_Description">
+        <p>
+          Evaluates Xpath query and report violation on all matching AST nodes. Current check allows
+          user to implement custom checks using Xpath.
+          Please read more about xpath syntax at
+          <a href="https://www.w3schools.com/xml/xpath_syntax.asp"> W3Schools Xpath Syntax</a>.
+        </p>
```

Comment:
Please specify explicitly that the check does nothing by default.

Reply:
done